src/components/TicketsItem.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+    <div class="ticket">
+        <div class="ticket__header">
+            <div class="ticket__price">
+                {{formatPrice}}
+            </div>
+            <div class="ticket__company-logo">
+                <!-- не осилил как забиндить путь до картинки вида "@/assets/images/" - vue ругался на пробел в названии картинки-->
+                <img :src="ticket.company.logo" :alt="ticket.company.name">
+            </div>
+        </div>
+        <div class="ticket__body">
+            <tickets-segments :segments="ticket.segments"/>
+        </div>
+    </div>
+</template>
+
+<script>
+import TicketsSegments from './TicketsSegments.vue'
+
+    export default {
+        components: {
+            TicketsSegments,
+        },
+
+        props: {
+            ticket: {
+                type: Object,
+                require: true,
+            },
+        },
+        computed: {
+            formatPrice() {
+                return new Intl.NumberFormat('ru-RU').format(this.ticket.price) + ' Р'
```

Comment:
Элегантно и просто!

Альтернативный вариант:
```suggestion
                return new Intl.NumberFormat('ru-RU', { style: "currency", currency: "RUB", maximumFractionDigits: 0 }).format(1000);
```